tests/framework/installer/ceph_installer.go

```diff
@@ -396,20 +396,20 @@ func (h *CephInstaller) InstallRook(namespace, storeType string, usePVC bool, st
 	logger.Infof("Installing rook on k8s %s", k8sversion)
 
 	startDiscovery := true
-	onamespace := namespace
+	onamespace := operatorNamespace
 	// Create rook operator
 	if h.useHelm {
 		// disable the discovery daemonset with the helm chart
 		settings := "enableDiscoveryDaemon=false"
 		startDiscovery = false
-		err = h.CreateRookOperatorViaHelm(namespace, settings)
+		err = h.CreateRookOperatorViaHelm(onamespace, settings)
```

Comment:
The helm chart is the only one that needs added support for multiple clusters, right? Looks like the `for` loop on line 428 should be here instead.